obal/data/modules/changelog.py

```diff
@@ -24,6 +24,18 @@ def en_locale():
             locale.resetlocale(locale.LC_TIME)
 
 
+def format_evr(epoch, version, release):
+    epoch = epoch.strip().replace('"', '')
+    version = version.strip().replace('"', '')
+    release = release.strip().replace('"', '')
+    evr = ""
+
+    if not '(none)' in epoch:
```

Comment:
The common order is `'(none)' not in epoch`.

Reply:
ah okay, thank you